### PR TITLE
Correctly detect log level when rendering logs as structured JSON

### PR DESCRIPTION
**Pull request by akshaymankar**
Fix for https://github.com/wireapp/wire-server/pull/1951#issuecomment-984479669

https://wearezeta.atlassian.net/browse/FS-62

## Checklist

 - [x] The **PR Title** explains the impact of the change.
 - [x] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
 - [x] **changelog.d** contains the following bits of information ([details](https://github.com/wireapp/wire-server/blob/develop/docs/developer/changelog.md)):
   - [x] A file with the changelog entry in one or more suitable sub-sections. The sub-sections are marked by directories inside `changelog.d`.

### CODE REVIEW

**Comment by akshaymankar** [comment]
I am off for the weekend. So please feel free to merge this if you approve.
If a release is made before merging this, we'll have to add a separate changelog, this is also fine IMO.

**Review by smatting — APPROVED** [review]

**Pull request merged**
